lib/api-fallback.ts

```
@@ -0,0 +1,33 @@
+// Simplified API without Edge runtime support
+export class APIFallback {
+  private static readonly NODE_ENDPOINT = '/api/ranking'
+  
+  static async fetchWithFallback(
+    params: URLSearchParams,
+    signal?: AbortSignal
+  ): Promise<Response> {
+    // Use Node.js endpoint only (Edge removed due to zlib incompatibility)
+    const response = await fetch(`${this.NODE_ENDPOINT}?${params.toString()}`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)